### DISCUSSION TITLE
Region of Interest Advice

### DISCUSSION THREAD

**Discussion by csmackey** [DeniseCaiLab/ezTrack#50] [Q&A]
Hello, thank you for this great tool! I started using ezTrack to analyze open field and elevated plus maze data and it feels like a lifesaver since I'm not in a neuroscience lab and the automation is great since I'm pretty much flying solo with my project. However, I've come across an issue I'd like to possibly get advice for. Specifically, for the EPM, some of the animals have a tendency to sit at the border between (for example) the center and closed arm. Free drawing the ROIs leads to a situation where runs will differ greatly (between 15-20%) in the percentage time spent in the center. 
I was wondering if there are any tools to place pre-made ROI maps onto the videos to keep them consistent or if you have any advice on ways to verify results to better validate ROI placement?

**Comment by ZachPenn** [comment]
Are you processing videos in batch, or individually?

**Reply by csmackey** [reply]
Individually, currently.

**Reply by ZachPenn** [reply]
In this case, I'm not sure what you mean by verifying ROI placement.  The ROI will be exactly where you draw it and is also overlaid on top of the reference image in the output (this is also done at the end of batch processing to help verify).  If you feel you aren't drawing accurately enough you can use the zooming tools.
